pkg/server/plugin/upstreamca/disk/disk_test.go

```diff
@@ -248,24 +235,20 @@ func testCSRResp(t *testing.T, resp *upstreamca.SubmitCSRResponse, expectCertCha
 func (s *DiskSuite) TestSubmitInvalidCSR() {
```

Comment:
Please add a test for an invalid sequence of bytes.

Reply:
Same as above.